cmd/register/register_util.go

```diff
@@ -402,8 +402,14 @@ func hydrateLaunchPlanSpec(configAssumableIamRole string, configK8sServiceAccoun
 	outputLocationPrefix := len(configOutputLocationPrefix) > 0
 	if assumableIamRole || k8sServiceAcct {
 		lpSpec.AuthRole = &admin.AuthRole{
-			AssumableIamRole:         configAssumableIamRole,
 			KubernetesServiceAccount: configK8sServiceAccount,
+			AssumableIamRole:         configAssumableIamRole,
```

Comment:
why should we set authrole?

Reply:
We have other places in flytectl where we set both auth role and security context. Will remove those separately as part of the clean up PR.